test/TEST-02-CRYPTSETUP/test.sh

```diff
@@ -86,10 +79,10 @@ EOF
 }
 
 test_cleanup() {
-    umount $TESTDIR/root/var 2>/dev/null
-    [[ -b /dev/mapper/varcrypt ]] && cryptsetup luksClose /dev/mapper/varcrypt
-    umount $TESTDIR/root 2>/dev/null
-    [[ $LOOPDEV ]] && losetup -d $LOOPDEV
+    umount $TESTDIR/root/var 2>/dev/null || true
+    [ ! -b /dev/mapper/varcrypt ] || cryptsetup luksClose /dev/mapper/varcrypt
+    umount $TESTDIR/root 2>/dev/null || true
+    [ -z "$LOOPDEV" ] || losetup -d $LOOPDEV || true
```

Comment:
It is assumed that these tests are run via `bash`, so I'm not sure I understand why `[[...]]` are being replaced by `[...]`.